### DISCUSSION TITLE
Teacher cannot be reverified.

### DISCUSSION THREAD

**Discussion by wenquan** [community/community#48618] [GitHub Education]
This is the message that I got after hours' trial and error. 

- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

I did not submit my school id since there is no date on it. So I uploaded a letter with letter head as well as detailed address of the University. Not sure Why this happens.

**Comment by Siunaus** [comment]
I uploaded my school id and it says 

Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
You are significantly more likely to be verified if you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

I even uploaded the letter of appointment given to me by the school, but still failed to pass the verification

**Comment by samus-aran** [comment]
Hi @wenquan 👋 

We're happy you're here. Here are a few tips that can help you with your application:

- For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. We encourage a Dated Student ID but we also understand that not all ID has a date - please feel free to us your current Student ID
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
